documents/PythonAPI/custom.css

```diff
@@ -0,0 +1,375 @@
+/*
+Dark Mode colors and other variables copied from "doxygen-awesome.css".
```

Comment:
Since we're already hosting a copy of Doxygen Awesome for our C++ API documentation, could we potentially just reuse that here as well, and remove the second copy from the MaterialX codebase?

https://github.com/AcademySoftwareFoundation/MaterialX/tree/main/documents/DoxygenAwesome

Reply:
Yes we can!

https://github.com/AcademySoftwareFoundation/MaterialX/pull/2038/commits/cea9275f6cecc44352c233fc9625e96e6772f74a

It required a couple of extra tweaks in `custom.css` to re-establish some of the styles from `alabaster.css`, but it works!

Reply:
Looks good, thanks @StefanHabel!